engines/bops_api/app/controllers/bops_api/v2/planning_applications_controller.rb

```diff
@@ -3,7 +3,36 @@
 module BopsApi
   module V2
     class PlanningApplicationsController < AuthenticatedController
-      validate_schema! "submission"
+      skip_before_action :authenticate, only: :determined
+
+      validate_schema! "submission", only: :create
+
+      def index
+        @pagy, @planning_applications = query_service.call
+
+        respond_to do |format|
+          format.json
+        end
+      end
+
+      def determined
+        # This endpoint is unauthenticated for public access
+        @pagy, @planning_applications = query_service(
+          planning_applications_scope.determined.by_determined_at_desc
+        ).call
+
+        respond_to do |format|
+          format.json
+        end
+      end
+
+      def show
+        @planning_application = planning_applications_scope.find(params[:id])
```

Comment:
Should we be validating that the param is numeric? I'm not sure whether it matters in all cases or if I'm just being overcautious:
```suggestion
        @planning_application = planning_applications_scope.find(Integer(params[:id]))
```